inventory/any.cf

```diff
@@ -603,6 +603,15 @@ bundle common cfe_autorun_inventory_aws
         scope => "namespace",
         if => isvariable("cfe_autorun_inventory_dmidecode.dmi[bios-vendor]");
 
+@if minimum_version(3.22.0)
```

Comment:
Do we have a feature registered for the function? Would be nice to be able to so if feature isreadable.